packages/api/functions/express/routes/program.js

```diff
@@ -50,6 +50,6 @@ programRouter.get('/:programId', wrapAsync(async (req, res) => {
 }))
 
 // project routes
-programRouter.use('/:programId/projects', projectRouter)
+// programRouter.use('/:programId/projects', projectRouter)
```

Comment:
Programs API has been moved to be under `/workspaces/:workspaceId/programs`, so as to avoid making the path of the project's resource very long (`/workspaces/:workspaceId/programs/:proramId/programs`), I removed this one and I have created one entry in `app.js`. 

`app.use('/programs/:programId/projects', projectRouter)`